compiler/passes/convert-uast.cpp

```diff
@@ -241,14 +328,46 @@ struct Converter {
 
   /// IndexableLoops ///
 
+  // In the uAST, loop index variables are represented as Decl, but in the
+  // old AST they are represented as expressions.
+  Expr* convertLoopIndexDecl(const uast::Decl* index) {
+    if (index == nullptr) return nullptr;
+
+    if (const uast::Variable* var = index->toVariable()) {
```

Comment:
How does this work for `for (i,j)`? How about `for (i, (j, k))`? I am guessing the answer is "it doesn't" and that's what the following `TODO` is, but just wanted to make sure if I'm following things right.

Reply:
Yep, the TODO will handle that later.